packages/frontend/src/features/omnibar/api/search.ts

```diff
@@ -1,4 +1,5 @@
 import { SearchFilters, SearchResults } from '@lightdash/common';
+import _ from 'lodash';
```

Comment:
can you import `omitBy` like this `import omitBy from 'lodash/omitBy'` because afaik build is not tree shaking at the moment and that's how we usually import now